### PR TITLE
#comemnt changed previewURL to downloadUrl changed version number - n…

### PR DESCRIPTION
**Pull request by niraliabhani**
…eeds updating nuget

### CODE REVIEW

**Comment by niraliabhani** [comment]
@leonatherton @KieranFrance changes made for csharp

File: ExampleUsage.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 using System;
+using System.Collections.Generic;
```
Comment (leonatherton) [inline ExampleUsage.cs]: Does this line need to be added?

Comment (RFoley) [reply ExampleUsage.cs]: The generics library is needed to use the Dictionary class - looks like it was accidentally missed out of a previous commit!

Comment (leonatherton) [reply ExampleUsage.cs]: Thanks @RFoley!

**Pull request merged**